src/sc/sidechain.h

```diff
@@ -33,6 +55,9 @@ class CSidechain {
     // last epoch for which a certificate have been received
     int lastEpochReferencedByCertificate;
```

Comment:
This should also be added in the output of getscinfo RPC cmd

Reply:
Agree, but as discussed AFK,  let's make it subject on subsequent PR